core/StdInt.carp

```diff
@@ -179,6 +179,7 @@
   (register copy (Fn [&Uint16] Uint16))
 
   (defn prn [a] (Uint16.str a))
+  (defn zero [] (from-long 0l))
```

Comment:
Will do! In general, I try to implement the absolute minimum necessary to avoid unwitting commit/PR scope creep....but following this principle blindly does lead to some obvious omissions like in this case!

Reply:
See commit 32b61fb!